### DISCUSSION TITLE
How do I remove Chinese language?

### DISCUSSION THREAD

**Discussion by rahaaatul** [F9y4ng/GreasyFork-Scripts#248] [问答专区(Question / Answer)]
How do I remove Chinese language from google search when I click on google from Bing search?

**Comment by F9y4ng** [comment]
Please temporarily in the following code:

https://github.com/F9y4ng/GreasyFork-Scripts/blob/44296798656417fc2e5ce678a785b3236e01c800/Google%20%26%20Baidu%20Switcher.user.js#L1325-L1326

Find and delete `hl=zh-CN&`.

I'll update it in next month's update.
